scripts/install

```diff
@@ -99,7 +99,7 @@ sudo chown -R $serviceuser: $final_path
 
 cd $final_path/programs/server/
 
-sudo npm install --production
+sudo -u $serviceuser npm install --production
```

Comment:
can you explain why this is necessary? I am just curious

Reply:
When you run npm install as a root, you have a warning message about running it as a root.
Since, we have a dedicated user, we should run npm using this user